pkg/machines/components/storagePools/storagePoolDelete.jsx

```diff
@@ -18,15 +18,56 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Modal } from 'patternfly-react';
+import { Button, Modal, OverlayTrigger, Tooltip } from 'patternfly-react';
 
-import { storagePoolId } from '../../helpers.js';
+import { getStorageVolumesUsage, storagePoolId } from '../../helpers.js';
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { storagePoolDeactivate, storagePoolUndefine, storageVolumeDelete } from '../../libvirt-dbus.js';
 import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+/*
```

Comment:
Can you use https://github.com/cockpit-project/cockpit/blob/master/pkg/machines/helpers.js#L612 instead of introducing these two new functions?
Feel free t o adjust it if needed, but basically we are checking the the exact same things when deleting volume, we should not introduce duplicate code.

Reply:
The function to which you refer just finds out if volumes are used as a disk. But at IRC conversation I explained that sometimes pool can be deleted without content even if volumes are used as disk. So there needs to be additional functions/wrappers because "volumes are used" != "pool cannot be deleted".
I updated these function a bit, renamed them so there is no confusion, but still:
Second function finds out if pool itself is referenced by some VM. In this case Pool cannot be deleted (nevertheless if user chooses with or without content deletion)
First function is mainly just a wrapper for the function which you mentioned, which converts result to boolean. It finds out if pool can be deleted **with it's content**.